docs/tutorials/index.md

```diff
@@ -120,7 +120,7 @@ The following tutorials show how visualization helps us in daily work, and bette
 The following tutorials explain how to develop machine learning applications running on embedded devices, such as the Raspberry Pi
```

Comment:
The following tutorials explain how to develop machine learning applications running on embedded devices, such as the Raspberry Pi: